waspc/src/Wasp/AppSpec/Valid.hs

```diff
@@ -68,24 +68,24 @@ validateDbIsPostgresIfPgBossUsed spec =
       ]
     else []
 
-validateAuthUserEntityHasCorrectFieldsIfEmailAndPasswordAuthIsUsed :: AppSpec -> [ValidationError]
-validateAuthUserEntityHasCorrectFieldsIfEmailAndPasswordAuthIsUsed spec = case App.auth (snd $ getApp spec) of
+validateAuthUserEntityHasCorrectFieldsIfUsernameAndPasswordAuthIsUsed :: AppSpec -> [ValidationError]
```

Comment:
Hehe, love this function name, I remember coming up with it and I just couldn't figure out anything shorter :D. But it works I guess!

Reply:
Haha, yeah it may win the award for longest function name I've seen in a while :D But it does indeed work!